src/v/pandaproxy/schema_registry/test/compatibility_avro.cc

```diff
@@ -250,3 +250,28 @@ SEASTAR_THREAD_TEST_CASE(test_avro_schema_definition) {
     pps::canonical_schema_definition avro_conversion{valid};
     BOOST_CHECK_EQUAL(expected, avro_conversion);
 }
+
+SEASTAR_THREAD_TEST_CASE(test_avro_schema_definition_custom_attributes) {
+    // https://github.com/redpanda-data/redpanda/issues/7274
+    // custom attributes supported only at field level
+    const auto avro_metadata_schema
+      = pps::sanitize_avro_schema_definition(
+          {R"({"type":"record","name":"foo","ignored_attr":true,"fields":[{"name":"bar","type":"float","extra_attr":true}]})",
+           pps::schema_type::avro})
+          .value();
+    pps::canonical_schema_definition expected{
+      R"({"type":"record","name":"foo","fields":[{"name":"bar","type":"float","extra_attr":true}]})",
```

Comment:
So we drop the custom attribute on the schema (`ignored_attr`) but not the one on the field (`extra_attr`)?

This isn't a blocker but more of a Q as to why dropping one and not the other is OK.

Reply:
I'm just going by what's written here: https://github.com/apache/avro/pull/1736